pkg/services/folder/folderimpl/folder.go

```diff
@@ -485,7 +516,9 @@ func (s *Service) Move(ctx context.Context, cmd *folder.MoveFolderCommand) (*fol
 	}
 
 	return s.store.Update(ctx, folder.UpdateFolderCommand{
```

Comment:
here i wonder instead of calling store, we need to call service, since we want also to update the folder in the dashboard table right now, to keep them consistent. And have userid which is the user updated folder.

Reply:
The only thing we change is the parent that does not exist in the legacy table, no?

Reply:
what we do need though is a check for permissions; I will add it